packages/wonder-blocks-button/src/components/button-core.tsx

```diff
@@ -377,17 +372,26 @@ const _generateStyles = (
                     borderRadius: 2,
                 },
             },
-            active: {
-                color: light ? fadedColor : activeColor,
+            focus: {
                 ":after": {
                     content: "''",
+                    // Since we are using a pseudo element, we need to manually
+                    // calculate the width/height and use absolute position to
+                    // prevent other elements from being shifted around.
                     position: "absolute",
-                    height: 2,
-                    width: `calc(100% - ${iconWidth}px)`,
-                    right: 0,
-                    bottom: -1,
+                    width: `calc(100% + 4px)`,
+                    height: `calc(100% - 4px)`,
```

Comment:
**suggestion:** Using `Spacing` as much as we can would be helpful. Specially with the theming work starting very soon :).

```suggestion
                    width: `calc(100% + ${Spacing.xxxSmall_4}px)`,
                    height: `calc(100% - ${Spacing.xxxSmall_4}px)`,
```